htcondor/publishers/publisher.py

```diff
@@ -88,19 +86,14 @@ def _condor_advertise(self, classads, collector_host=None,
             else:
                 collector_host = 'default'
                 collector = htcondor.Collector()
-            self.logger.info('Advertising %s classads to collector_host %s' % (
-                self.classad_type, collector_host))
+
+            self.logger.info(f"Advertising {self.classad_type} classads to collector_host {collector_host}")
             collector.advertise(ads, update_ad_command, True)
         except Exception:
```

Comment:
since this looks like a defensive except, can you add # pragma: no cover to the except Exception: line?